<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cameraSettings2d</class>
 <widget class="QDialog" name="cameraSettings2d">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kinect Touch Settings</string>
  </property>
  <widget class="glwidget2d" name="glImage" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>68</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Depth</string>
   </property>
  </widget>
  <widget class="glwidget2d" name="glBlobs" native="true">
   <property name="geometry">
    <rect>
     <x>687</x>
     <y>40</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="methodToUse">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Background</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Plane</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>34</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Method : </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="commonSettings">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>221</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Common Processing  Settings</string>
   </property>
   <widget class="QSpinBox" name="minDistance">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-6000</number>
    </property>
    <property name="maximum">
     <number>6000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxDistance">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-6000</number>
    </property>
    <property name="maximum">
     <number>6000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="minBlobSize">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>33</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Distance</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>73</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Distance</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>113</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Blob Size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoneButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>154</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Valid Zone</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>200</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Dilate     -&gt;     Erode      -&gt;   Dilate</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="dilate0">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>230</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="erode">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="dilate1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="refineMask">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Refining Mask</string>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>548</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Working on :</string>
   </property>
  </widget>
  <widget class="QLabel" name="processLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>501</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Normal processing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Progress :</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>363</x>
     <y>10</y>
     <width>31</width>
     <height>571</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>10</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Final Blob Image</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="backSettings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>300</width>
     <height>220</height>
    </rect>
   </property>
   <property name="title">
    <string>Background Method Settings</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="calBack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>291</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Background</string>
    </property>
   </widget>
   <widget class="glwidget2d" name="imgBack" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Background : </string>
    </property>
   </widget>
   <widget class="glwidget2d" name="imgMaskBack" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mask :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Accepted Variance of Pixels : </string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="variance">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>157</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>994.100000000000023</double>
    </property>
    <property name="maximum">
     <double>996.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="planeSettings">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>350</y>
     <width>300</width>
     <height>220</height>
    </rect>
   </property>
   <property name="title">
    <string>Plane Method Settings</string>
   </property>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mask :</string>
    </property>
   </widget>
   <widget class="glwidget2d" name="imgMaskPlane" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="glwidget2d" name="imgPlane" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Plane : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="calcPlane">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>291</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Plane</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="Instructions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>This text will display instructions when needed. </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="matchIandD">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Match Image and Depth</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glwidget2d</class>
   <extends>QWidget</extends>
   <header>glwidget2d.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
